client/jetpack-cloud/components/sidebar/index.tsx

```diff
@@ -54,6 +56,9 @@ const JetpackCloudSidebar = ( {
 		siteId ? getJetpackAdminUrl( state, siteId ) : null
 	);
 
+	useQueryJetpackPartnerPortalPartner();
+	const canAccessJetpackManage = useSelector( hasActivePartnerKey );
```

Comment:
Can we replace this with `has_jetpack_partner_access` props from the currentUser object? This way, we do not need to query the API.